dokka-subprojects/plugin-base/src/main/kotlin/org/jetbrains/dokka/base/renderers/html/htmlPreprocessors.kt

```diff
@@ -104,6 +104,12 @@ public class StylesInstaller(private val dokkaContext: DokkaContext) : PageTrans
 
 public object AssetsInstaller : PageTransformer {
     private val imagesPages = listOf(
+        // ui-kit assets
```

Comment:
For now there is no way to register all content from the assets directory so we have to register it file by file